packages/dcos-integration-test/extra/conftest.py

```diff
@@ -31,17 +34,39 @@ def dcos_api_session(dcos_api_session_factory):
     return api
 
 
+def pytest_cmdline_main(config):
+    user_outside_cluster = True
+    if os.path.exists('/opt/mesosphere/bin/dcos-shell'):
+        user_outside_cluster = False
+
+    if user_outside_cluster and config.option.env_help:
+        print(env_helper.HELP_MESSAGE)
+        sys.exit()
+
+    if user_outside_cluster and not config.option.help and not config.option.collectonly:
+        env_vars = env_helper.get_env_vars()
+        if config.option.dist == 'no':
+            config.option.dist = 'load'
```

Comment:
What does this setting do? Can I set it to no myself?

Reply:
will address these in another PR and merge this so people can try it out